cluster/gce/config-test.sh

```diff
@@ -146,22 +146,34 @@ CONTROLLER_MANAGER_TEST_LOG_LEVEL="${CONTROLLER_MANAGER_TEST_LOG_LEVEL:-$TEST_CL
 SCHEDULER_TEST_LOG_LEVEL="${SCHEDULER_TEST_LOG_LEVEL:-$TEST_CLUSTER_LOG_LEVEL}"
 KUBEPROXY_TEST_LOG_LEVEL="${KUBEPROXY_TEST_LOG_LEVEL:-$TEST_CLUSTER_LOG_LEVEL}"
 
+# TODO: change this and flex e2e test when default flex volume install path is changed for GCI
+# Set flex dir to one that's readable from controller-manager container and writable by the flex e2e test.
+CONTROLLER_MANAGER_TEST_VOLUME_PLUGIN_DIR=""
+if [[ "${MASTER_OS_DISTRIBUTION}" == "gci" ]]; then
+    CONTROLLER_MANAGER_TEST_VOLUME_PLUGIN_DIR="--flex-volume-plugin-dir=/etc/srv/kubernetes/kubelet-plugins/volume/exec"
```

Comment:
how do you mean, the controller manager container cannot read that dir by default?

/etc/srv/kubernetes/ seems to me to be the least bad place to put it for now since it is used to store various persistent things like auth certs and kubeconfigs.